### ISSUE TITLE
Unable to use keyboard shortcut if toolbar is hidden

### ISSUE THREAD

**Issue by iseulde** [WordPress/gutenberg#3035]
## Issue Overview
After #2960, pressing the meta key (cmd/ctrl) still fails to focus the toolbar when it is hidden by typing.

## Steps to Reproduce (for bugs)
1. Select a block.
2. Type inside the block so that the toolbar hides.
3. Press cmd on Mac or ctrl elsewhere.
4. Nothing happens.

## Expected Behavior
I expected the toolbar to show and the first item the receive focus.

## Related Issues and/or PRs
* #2960.

**Comment by iseulde** [comment]
@youknowriad That's probably a downside of doing this in the toolbar component, which removes the event listeners when it unmounts.

**Comment by youknowriad** [comment]
This will be closed once #2998 land
